builder/builder.go

```diff
@@ -0,0 +1,82 @@
+package builder
+
+import (
+	"archive/tar"
+	"bytes"
+	"context"
+	"github.com/docker/docker/api/types"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/docker/docker/client"
+)
+
+// Builds Docker Image for Code Runner.
+func BuildCodeRunnerImage() error {
+
+	buildContext := context.Background()
+	dockerClient, err := client.NewEnvClient()
+	if err != nil {
+		log.Fatalf("error in creating a docker client: %v", err)
+	}
+
+	// Create a tar for build context.
+	tarBuffer := new(bytes.Buffer)
+	writer := tar.NewWriter(tarBuffer)
+	defer func() {
+		err = writer.Close()
+		if err != nil {
+			log.Fatal(err)
+			return
+		}
+	}()
+
+	dockerFilepath := "Dockerfile"
```

Comment:
no magic strings!!
Maybe have it as a constant that is also used by the docker file creator. That way, it's consistent.

Reply:
Done